src/main/java/bio/terra/service/dataset/Dataset.java

```diff
@@ -159,6 +159,15 @@ public Dataset projectResourceId(UUID projectResourceId) {
         return this;
     }
 
+    public List<StorageResource> getStorage() {
```

Comment:
It's really more the setter. That allows us to use fluent syntax on the Dataset object when we're building a dataset (e.g.
``` new Dataset().storage(List.of(...)).name("some name")...```)

Which we can't do if we have to edit the storage object inside of dataset summary (e.g. ``` new Dataset().getSummary().storage(List.of(...)).schema(...)...``` would  fail since `schema()` is on the `Dataset` object but `storage()` returns a `DatasetSummary`

Reply:
Oh I see! Thanks for clarifying